src/test/resources/testVariable.xml

```diff
@@ -215,4 +216,30 @@
     </CONSTRAINT>
   </INPUT>
 
+  <INPUT>
+    <NAME>password_master</NAME>
+    <DESCRIPTION>Some password</DESCRIPTION>
+    <CONSTRAINT>
+      <TYPE>masterPassword</TYPE>
+      <PASSWORDHASH>linux-shadow-sha512</PASSWORDHASH>
+      <USES>password_slave1, password_slave2, not_check_on_name_is_done_here</USES>
```

Comment:
no, the parser don't know what is the semantic of the use, it is field (UI) dependant. We could imagine that in some case, we only want to look for field on the same section, and on other case, we won't. I think. 
Perhaps we could at least add a sanity check after the parsing, to avoid typo ?